apps/sync/js/adapters/history.js

```diff
@@ -0,0 +1,98 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
```

Comment:
The example data records have updated [here](https://github.com/mozilla-b2g/gaia/pull/31896/files#diff-4d73ff90f2479c9b5d1f53457e85e25dR204). It includes "deleted: true" as well.